uber/model_checks.py

```diff
@@ -37,6 +37,30 @@ def has_email_address(account):
 
 
 @validation.Group
```

Comment:
This will almost certainly be a problem.  The issue is that admins will often create groups which get tables which aren't dealers per se.  The main use case is bands, who have groups created for them by admins but don't have these fields filled out.

To give you some idea of how often this happens, here's a query from last year's database:

```python
>>> session.query(Group).filter(Group.tables > 0, Group.wares == '').count()
50
```

I wouldn't be comfortable with this change until we make sure that the affected admins are okay with it.  In the case of the ``wares`` column I could see still requiring it, but some of the other fields like ``website`` probably need to not be required on the admin side.